File: app/src/main/java/maw/org/wedding/android/home/HomeFragment.java
package maw.org.wedding.android.home;

import android.app.Fragment;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import maw.org.wedding.R;

public class HomeFragment extends Fragment {

    public static Fragment newInstance() {
        return new HomeFragment();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {

        View rootView = inflater.inflate(R.layout.home_fragment_layout, container, false);
        return rootView;
    }
}

File: app/src/main/kotlin/maw/org/wedding/home/HomeFragment.kt
package maw.org.wedding.home

import android.app.Fragment
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import maw.org.wedding.R

class HomeFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.home_fragment_layout, container, false)
    }

    companion object {
        fun newInstance(): Fragment {
            return HomeFragment()
        }
    }
}
